app/controllers/invite_codes_controller.rb

```diff
@@ -0,0 +1,15 @@
+class InviteCodesController < ApplicationController
+  def index
+    @invite_codes = fetch_invite_codes
+    respond_to do |format|
+      format.html
+      format.turbo_stream { render turbo_stream: turbo_stream.replace("invite_codes", partial: "invite_codes") }
+    end
+  end
+
```

Comment:
Mentioned this in a comment below, but I think we could break this controller into:

```rb
def index
  @invite_codes = InviteCode.all
end

def create
  InviteCode.generate!
  redirect_back_or_to invite_codes_path, notice: "Code generated"
end
```

Reply:
[dee9af2](https://github.com/maybe-finance/maybe/pull/1163/commits/dee9af253bfc556578ca75bfa2d98b89cdcb05e0)